### PR TITLE
Bot API 6.0 backported from grammyjs/types

### PR DESCRIPTION
**Pull request by MKRhere**
Signed-off-by: Muthu Kumar <muthukumar@thefeathers.in>

### CODE REVIEW

File: message.d.ts [hunk 1]
```diff
@@ -184,21 +184,24 @@ export namespace Message {
...
   }
-  export interface VoiceChatParticipantsInvitedMessage extends ServiceMessage {
-    /** Service message: new participants invited to a voice chat */
-    voice_chat_participants_invited: VoiceChatParticipantsInvited;
+  export interface VideoChatParticipantsInvitedMessage extends ServiceMessage {
+    /** Service message: new participants invited to a video chat */
+    video_chat_participants_invited: VideoChatParticipantsInvited;
+  }
+  export interface WebAppDataMessage extends ServiceMessage {
+    web_app_data: WebAppData;
```
Comment (KnorpelSenf) [inline message.d.ts:205]: Missing docs string

File: message.d.ts [hunk 2]
```diff
@@ -221,10 +224,10 @@ export type ServiceMessageBundle =
...
   | Message.SuccessfulPaymentMessage
   | Message.SupergroupChatCreated
-  | Message.VoiceChatScheduledMessage
-  | Message.VoiceChatStartedMessage
-  | Message.VoiceChatEndedMessage
-  | Message.VoiceChatParticipantsInvitedMessage;
+  | Message.VideoChatScheduledMessage
+  | Message.VideoChatStartedMessage
+  | Message.VideoChatEndedMessage
+  | Message.VideoChatParticipantsInvitedMessage;
```
Comment (KnorpelSenf) [inline message.d.ts]: Missing addition of `WebAppDataMessage`

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
Almost done.

File: message.d.ts (hunk 1 above)
Comment (MKRhere) [reply message.d.ts:205]: Fixed

File: message.d.ts (hunk 2 above)
Comment (MKRhere) [reply message.d.ts]: Fixed

**Review by KnorpelSenf — APPROVED** [review]
LGTM, thanks!

**Pull request merged**